remote/web/package.json

```
@@ -6,25 +6,26 @@
 		"@microsoft/1ds-core-js": "^3.2.13",
 		"@microsoft/1ds-post-js": "^3.2.13",
 		"@vscode/iconv-lite-umd": "0.7.0",
+		"@vscode/tree-sitter-wasm": "^0.0.2",
 		"@vscode/vscode-languagedetection": "1.0.21",
-		"@xterm/addon-clipboard": "0.2.0-beta.34",
-		"@xterm/addon-image": "0.9.0-beta.51",
-		"@xterm/addon-search": "0.16.0-beta.51",
-		"@xterm/addon-serialize": "0.14.0-beta.51",
-		"@xterm/addon-unicode11": "0.9.0-beta.51",
-		"@xterm/addon-webgl": "0.19.0-beta.51",
+		"@xterm/addon-clipboard": "0.2.0-beta.35",
+		"@xterm/addon-image": "0.9.0-beta.52",
+		"@xterm/addon-search": "0.16.0-beta.52",
+		"@xterm/addon-serialize": "0.14.0-beta.52",
+		"@xterm/addon-unicode11": "0.9.0-beta.52",
+		"@xterm/addon-webgl": "0.19.0-beta.52",
+		"@xterm/xterm": "5.6.0-beta.52",
 		"@xterm/xterm": "5.6.0-beta.51",
-		"he": "^1.2.0",
+    "he": "^1.2.0",
```

looks like the indentation changed from tabs to spaces on this line
```suggestion
		"he": "^1.2.0",
```

---

fixed in https://github.com/posit-dev/positron/pull/4655/commits/0ebafbf76c1da36368b1ca2071c8f5058e1d3086!